examples/encoding/plot_continuous_encoder.py

```diff
@@ -0,0 +1,114 @@
+interpreted as a measure of functional connectivity. These inputs could also
+be stimulus features, such as spectral features of sound. In this case, model
+coefficients would be measures of receptive field properties for the channel.
+
+Estimation of spectro-temporal and spatio-temporal receptive fields using
+modeling with continuous inputs is described in:
+
+    Theunissen, F. E. et al. Estimating spatio-temporal receptive
+    fields of auditory and visual neurons from their responses to
+    natural stimuli. Network 12, 289-316 (2001).
+
+    Willmore, B. & Smyth, D. Methods for first-order kernel
+    estimation: simple-cell receptive fields from responses to
+    natural scenes. Network 14, 553-77 (2003).
+"""
+# Authors: Chris Holdgraf <choldgraf@gmail.com>
+#
+# License: BSD (3-clause)
+
+import matplotlib.pyplot as plt
+
+import mne
+import numpy as np
+from mne.datasets import sample
+from mne.encoding import SampleMasker, get_coefs, get_final_est, FeatureDelayer
+from sklearn.pipeline import make_pipeline
+from sklearn.linear_model import Ridge
+from sklearn.preprocessing import StandardScaler
+
+###############################################################################
+# Load and preprocess data
+data_path = sample.data_path()
+raw_fname = data_path + '/MEG/sample/sample_audvis_filt-0-40_raw.fif'
+raw = mne.io.read_raw_fif(raw_fname, preload=True).pick_types(
+    meg=False, stim=False, eeg=True).filter(1, None, method='iir')
+seed_elec = 'EEG 011'
+ix_seed = mne.pick_channels(raw.ch_names, [seed_elec])[0]
+ix_targets = np.setdiff1d(range(len(raw.ch_names)), [ix_seed])
+fig = raw.plot_sensors()
+colors = np.zeros(len(raw.ch_names)).astype(bool)
+colors[ix_seed] = True
+colors = np.where(colors, 'r', 'k')
+plt.setp(fig.axes[0].collections[0], facecolors=colors, edgecolors=colors)
+fig.suptitle('Sensor positions (seed = red)')
+
+# Set up encoding model variables
+X = raw[:][0][ix_targets]
+y = raw[:][0][ix_seed]
+n_targets, n_times = X.shape
+
+###############################################################################
+# Fit the model / Make predictions
+# Define model preprocessing and parameters
+ix_split = int(n_times * .998)  # Using subset of data so it fits in a plot
+ix_tr = np.arange(n_times)[:ix_split]
+ix_tt = np.arange(n_times)[ix_split:]
+
+# This creates time-lagged versions of the data
+delay_step = .01
+delayer = FeatureDelayer(sfreq=raw.info['sfreq'],
+                         delays=np.arange(-.1, .1, delay_step))
+info_new = raw.copy().drop_channels([seed_elec]).info.copy()
+info_new['sfreq'] = 1. / delay_step
+
+# This is the final estimator that will be called after masking data
+pipe_est = make_pipeline(StandardScaler(), delayer, Ridge(alpha=1.))
+# This allows us to use a subset of indices for training / prediction
+pipe_full = SampleMasker(pipe_est, ixs=ix_tr, ixs_pred=ix_tt)
+
+# Fit / predict w/ the model
+pipe_full.fit(X.T, y)
```

Comment:
I kept the SampleMasker input to be the same shape as the scikit learn default (samples x features). Otherwise I thought it'd get confusing when using a regular sklearn estimator vs. an MNE estimator. WDYT?